MoreLinq/RandomSubset.cs

```diff
@@ -23,6 +23,45 @@ namespace MoreLinq
 
     public static partial class MoreEnumerable
     {
+        /// <summary>
+        /// Returns a sequence of random elements from the original sequence.
+        /// </summary>
+        /// <typeparam name="T">The type of source sequence elements.</typeparam>
+        /// <param name="sequence">
+        /// The sequence from which to return random elements.</param>
+        /// <returns>
+        /// A random sequence of elements in random order from the original
+        /// sequence.
+        /// </returns>
+        public static IEnumerable<T> Shuffle<T>(this IEnumerable<T> sequence)
```

Comment:
Add blank line for clarity.

---
PS I understand we need to do this consistently elsewhere in the source code, and we'll get round to it eventually, but meanwhile this also renders it consistent with this source file.